lib/elementSearch.js

```diff
@@ -10,6 +10,7 @@ function match(text, options = {}, ...args) {
   const get = async (tagName = '*') => {
     let elements;
     const textSearch = function (selectorElement, args) {
+      const searchText = args.text.toLowerCase().trim();
```

Comment:
Do we need a null check here?

Reply:
we do a type assertion at the beginning of method.. https://github.com/getgauge/taiko/blob/2b9e3dc11c887b0bfb5560a043600e742c3617c2/lib/elementSearch.js#L9